collector/pkg/component/analyzer/cpuanalyzer/cpu_analyzer.go

```diff
@@ -59,20 +65,34 @@ func NewCpuAnalyzer(cfg interface{}, telemetry *component.TelemetryTools, consum
 	}
 	ca.cpuPidEvents = make(map[uint32]map[uint32]*TimeSegments, 100000)
 	ca.tidExpiredQueue = newTidDeleteQueue()
-	ca.javaTraces = make(map[string]*TransactionIdEvent, 100000)
+	ca.javaTraces = make(map[JavaTracesKey]*TransactionIdEvent, 100000)
 	newSelfMetrics(telemetry.MeterProvider, ca)
 	return ca
 }
 
 func (ca *CpuAnalyzer) Start() error {
-	// Disable receiving and sending the profiling data by default.
+	ca.cleanerTicker = time.NewTicker(time.Duration(ca.cfg.JavaTraceDeleteInterval) * time.Second)
+	go func() {
+		for range ca.cleanerTicker.C {
```

Comment:
`cleanerTicker.Stop()` cannot close the ticker's channel, so this goroutine won't exit. You can use `stopProfileChan` to exit this goroutine.